about.html

```diff
@@ -0,0 +1,147 @@
+      <li class="sessionbar px-4">My Page</li>
+      <li class="sessionbar px-4">Logout</li>
+    </ul>
+  </div>
+
+  
+  <div class=" secondary-nav">
+    <ul class="d-flex px-4 mt-2">
+      <li>
+        <a href="index.html"><img class="navlogo " src="./capstoneimgs/logo.png" alt="world-music-festival-logo"></a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="index.html">Home</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Program</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Performers</a>
+      </li>
+      <li class="sec-nav-item">
+        <a class="sec-nav-link px-4" href="#">Partners</a>
+      </li>
+      <li class="sec-nav-item-btn">
+        <button class="sec-nav-link-btn px-4">Campaign</button>
+      </li>
+    </ul>
+  </div>
+
+        <header id="header">
+          <div id="header-nav-bar">
+            <nav id="burguer">
+              <div id="mobile-menu">
+                <img id= "menu-burguer" src="./capstoneimgs/burguer.svg" alt="" class="menu-icon p-3">
+                <img id= "close-menu" src="./capstoneimgs/x-button-icon-20.jpg" alt="" class="close-menu-icon p-3">
+              </div>
+              <div id="menu-container">
+                <ul class="menu">
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center text-danger" >
+                    <a href="index.html">Home</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center"> 
+                    <a href="#program">Program</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#join">Join</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#sponsor">Sponsor</a>
+                  </li>
+                  <li class ="nav-li1 p-2 w-100 d-flex justify-content-center">
+                    <a href="#news">News</a>
+                  </li>
+              </ul>
+              </div> 
+            </nav>
+          </div>                     
+        </header>
+  </section>
+
+
+      <section id="headline">
+        <p class="headline-hello">"Hello! World Music Fans!"</p>
+          <h1 class="headline-title">INTI RAYMI MUSIC FESTIVAL 2023</h1>
+        <p class="headline-p">Inti Raymi Music Festival brings together some of the most recognized musicians in Ecuador to perform for you all in the Mitad del Mundo City's Central Plaza. Come experience this celebration of thousands of years of tradition in cultural heritage.</p>
+        <p class="contact">Please contact us per email for any further questions 
+          about Inti Raymi Music Festival 2023!</p>
+        <address>intiraymifestivalecuador@somemail.not</address>
+      </section>
```

Comment:
You did a great job working on the design of your project! 💯 However, there are some changes that you can do in order to improve this headline section!

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/226496756-82afcb3c-8dd5-411d-aef8-1b90f5c63ac7.png) | ![image](https://user-images.githubusercontent.com/77069011/226496674-dffc691c-27db-4333-babb-052babd8b272.png) |


- [ ]  Please consider including a background image, or at least add a light gray background color, this would make your project look better and match the design. 🤓 

- [ ] Also, please consider correcting the spacing of the title, and its font-size, at the moment is way too big compared with the design! 🤓 

- [ ] Similarly, please consider fixing the size of the elements of this section, the description paragraph is way too large, and the elements have a lot of blank space in between, try to make the section fit in one screen (100vh). 🤓 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/226496756-82afcb3c-8dd5-411d-aef8-1b90f5c63ac7.png) | ![image](https://user-images.githubusercontent.com/77069011/226496871-dbe7eebd-c6ec-4966-9b6e-d707f8d458af.png) |

Reply:
Great! You're right I've fixed it!

Reply:
- [ ] This issue is not well-resolved :x: